sync-diff-inspector/shard-diff.md

```diff
@@ -118,3 +122,95 @@ fix-sql-file = "fix.sql"
     password = "123456"
     instance-id = "target-1"
 ```
+
+## Use `table-rules` for configuration
+
+You can use `table-rules` for configuration when there are a large number of upstream sharded tables and the naming rules of all sharded tables are regular as shown below:
+
+![shard-table-replica-2](/media/shard-table-replica-2.png)
+
+Below is a complete example of the sync-diff-inspector configuration.
+
+```
+# Diff Configuration.
+######################### Global config #########################
+# The log level. You can set it to "info" or "debug".
+log-level = "info"
+
+# sync-diff-inspector divides the data into multiple chunks based on the primary key,
+# unique key, or the index, and then compares the data of each chunk.
+# Compares data in each chunk. Uses "chunk-size" to set the size of a chunk.
```

Comment:
```suggestion
# Compare data in each chunk. Use "chunk-size" to set the size of a chunk.
```

Reply:
Adding "s" is correct. See [community/code-comment-style.md at ffedb17a569691800d901c5ee258b1cc4bf166ac · pingcap/community](https://github.com/pingcap/community/blob/ffedb17a569691800d901c5ee258b1cc4bf166ac/contributors/code-comment-style.md#language-for-a-good-comment):

When used for description, comments should be descriptive rather than imperative

- Opens the file (Right)
- Open the file (Wrong)